configs/components/rubygem-puppet.rb

```diff
@@ -1,6 +1,16 @@
 component 'rubygem-puppet' do |pkg, settings, platform|
-  pkg.version "6.20.0"
-  pkg.md5sum 'b1a6f244663f04075bafb1d36eede31c'
+  # Projects may define a :rubygem_puppet_version setting, or we use 6.20.0 by default:
+  version = settings[:rubygem_puppet_version] || '6.20.0'
+  pkg.version version
+
+  case version
+  when '7.3.0'
+    pkg.md5sum '82857bbc5a75c3b6e49ce15c18959504'
+  when '6.20.0'
```

Comment:
This is going to force everyone else to configure this - I think making `6.20` the default would be better.

Reply:
^ It is the default

```
version = settings[:rubygem_puppet_version] || '6.20.0'
```

Reply:
Oh weird...sorry, we don't usually do it that way. Didn't even look!